lib/licence_location_identifier.rb

```diff
@@ -1,30 +1,36 @@
-require 'location_identifier'
+class LicenceLocationIdentifier
+  AUTHORITY_TYPES = %w(DIS LBO UTA CTY LGD MTD COI).freeze
+  ALLOWED_REQUESTED_AREA_TYPES = %(CTY).freeze
+
+  def self.find_slug(areas, requested_area_type = "")
```

Comment:
Looks like it's called with `@licence_details.offered_by_county?` as the second param. Is that a `boolean`?